_posts/2019-01-10-our-okr-process.md

```diff
@@ -77,7 +75,7 @@ So before the first big OKR-Workshop nobody can exactly know what he or she will
 ### OKR-Workshop II
 
 After the first OKR-Workshop people who attended the workshop start informing all teams about the temporary results of that workshop:
-What ended up on the Kill-List? What might a team work on? How many resources are needed for that new Initiative? At this point, nothing is set in stone and no final decision was taken.
+What ended up on the final list? What might a team work on? How many people are needed for that new Initiative? At this point, nothing is set in stone and no final decision was taken.
```

Comment:
How is that you replace `Kill-List` with `final list` here? Kill-list is something that we won't work on. Should we really call it `final list` if further on you are saying that `no final decision is taken`

Reply:
I've got the feedback to not disclose too many "internal" wordings and names for things. But your right, now it was misleading. I changed it. 👍